### ISSUE TITLE
Allow generating a changelog, starting with configured tag.

### ISSUE THREAD

**Issue by ctron** [convco/convco#30]
**Is your feature request related to a problem? Please describe.**

When creating a changelog for an existing project (not using conventional commits so far) it creates a changelog with empty sections.

**Describe the solution you'd like**

* Allow to disable generating empty releases until the first release with content was found (might be a reasonable default)
* Allow to configure the starting point manually

**Describe alternatives you've considered**

None.

**Additional context**

**Comment by hdevalke** [comment]
Good point, having a release without changelog information is not that usefull, unless one would want to manually change the resulting file. Maybe we could add a flag `--skip-empty`. If you use conventional commits, there should not be any empty sections, unless you configure to only show features and have a release with only fixes.

**Comment by hdevalke** [comment]
I made a pull request to skip _empty_ releases. 

> Allow to configure the starting point manually
Here I don't know if it is worth to add it in config as you could use `vx.y.z..` in your CI to create the changelog excluding everything  `<= x.y.z`

**Comment by ctron** [comment]
> Here I don't know if it is worth to add it in config as you could use vx.y.z.. in your CI to create the changelog excluding everything <= x.y.z

That is good to know. I will try that, thanks!
